chaos_genius/settings.py

```diff
@@ -83,8 +84,8 @@ def _make_bool(val: Union[str, bool]) -> bool:
     if enabled_time_range not in TIME_RANGES_BY_KEY.keys():
         raise ValueError(f"Values in DEEPDRILLS_ENABLED_TIME_RANGES must be one of {', '.join(TIME_RANGES_BY_KEY.keys())}. Got: {enabled_time_range}.")
 TIMEZONE = os.getenv('TIMEZONE', default='UTC')
-if TIMEZONE not in SUPPORTED_TIMEZONES:
-    raise ValueError(f"Value of TIMEZONE must be one of {', '.join(SUPPORTED_TIMEZONES)}. Got: {TIMEZONE}.")
+if TIMEZONE not in SUPPORTED_TIMEZONES and TIMEZONE not in pytz.all_timezones:
+    raise ValueError(f"Invalid Timezone Provided. Value of TIMEZONE must be one of [{', '.join(SUPPORTED_TIMEZONES)}] or {pytz.all_timezones}. Got: {TIMEZONE}.")
```

Comment:
This is a very big list. Looks messy in the terminal and is difficult to search through. Can we link to the documentation instead?

Although, if the docs aren't updated, this is fine for now.